Update design-guidance-for-replicated-tables.md (#118482)
Azure/azure-content: articles/synapse-analytics/sql-data-warehouse/design-guidance-for-replicated-tables.md

SELECT TOP 1 * FROM [ReplicatedTable]
```

> To monitor the rebuild process, you can use 'sys.dm_pdw_exec_requests,' where the command will start with 'BuildReplicatedTableCache' as shown below

Reviewer: ```suggestion
To monitor the rebuild process, you can use [sys.dm_pdw_exec_requests](/sql/relational-databases/system-dynamic-management-views/sys-dm-pdw-exec-requests-transact-sql?view=azure-sqldw-latest&preserve-view=true), where the `command` will start with 'BuildReplicatedTableCache'. For example:
```

```sql
-- Monitor Build Replicated Cache
SELECT *
FROM sys.dm_pdw_exec_requests
WHERE command like 'BuildReplicatedTableCache%'
```

> [!TIP]

> > [Table size queries](https://learn.microsoft.com/azure/synapse-analytics/sql-data-warehouse/sql-data-warehouse-tables-overview#table-size-queries) can be used to verify which table(s) have a replicated distribution policy and whose size is more than 2 GB.

Reviewer: ```suggestion
> [Table size queries](/azure/synapse-analytics/sql-data-warehouse/sql-data-warehouse-tables-overview#table-size-queries) can be used to verify which table(s) have a replicated distribution policy and which are larger than 2 GB.
```
